File: lab2/FilmsBrowser/app/src/main/java/com/example/filmsbrowser/model/Film.kt
package com.example.filmsbrowser.model

import com.google.firebase.database.Exclude

class Film() {
    @Exclude
    var id: String? = null

    lateinit var name: String
    lateinit var categories: List<String>
    lateinit var description: String

    constructor(name: String, categories: List<String>, description: String) : this() {
        this.name = name
        this.categories = categories
        this.description = description
    }
}

File: lab1/CoolFilms/CoolFilms/Model/Film.swift
import Foundation

class Film: Identifiable {
    var id: String?
    var name: String
    var categories: [String]
    var description: String

    init(name: String, categories: [String], description: String) {
        self.name = name
        self.categories = categories
        self.description = description
    }
}
